### PR TITLE
remove bizzare circular deps #260

### PR DESCRIPTION
**Pull request by calvinmetcalf**
remember this commit a477a21474c0db82e34ee0bb1c0ff3b05dae1e41 ? yeah turns out it's a breaking change, but this didn't actually break anything in those 2 years, plus there has been a breaking change since then we can probably take it out.

### CODE REVIEW

File: readable.js [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
 exports.PassThrough = require('./lib/_stream_passthrough.js');
 
-if (!process.browser && process.env.READABLE_STREAM === 'disable' && Stream) {
-  module.exports = Stream;
+if (!process.browser) {
+  var Stream = require('st' + 'ream');
+  if (process.env.READABLE_STREAM === 'disable' && Stream) {
+    module.exports =  Stream;
+  }
 }
```
Comment (calvinmetcalf) [inline readable.js]: yes

File: package.json [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "pipe"
   ],
   "browser": {
-    "util": false
+    "util": false,
+    "./readable.js": "./readlable-browser.js"
```
Comment (mcollina) [inline package.json]: this is probably a typo.

Comment (calvinmetcalf) [inline package.json]: trying to do too many things at once thanks

**Comment by calvinmetcalf** [comment]
closing in favor of #269

**Pull request closed without merging**